java/src/jmri/jmrit/operations/router/JmritOperationsRouterBundle.properties

```diff
@@ -29,6 +29,7 @@ RouterDestination       = destination
 RouterTrainCanNotStaging = Train ({0}) can''t transport car ({1}) out of staging ({2}) directly to ({3}, {4})
 RouterAttemptStaging    = Try to find a route for car ({0}) through staging
 RouterNTrains        = Routing using {0} trains destination ({1}, {2}) begins
+RouterStagingExcluded	= Exclude staging ({0}, {1}) is has load modification enabled
```

Comment:
English is not my native language and I might be wrong. But `is has` in this sentence looks strange to me.

Reply:
Thanks for catching, fixed